api/model.ts

```diff
@@ -8,7 +8,7 @@ export enum SortOrder {
 }
 
 export type Sort<T> = {
-    field: keyof T
+    field: keyof T | string
```

Comment:
@kpudlik I want to sort by ordinal which is nested `lastTransactionRef.ordinal`
I set there also a `string` type but maybe you have a better idea?

Reply:
Typescript doesn't allow that :<

You can technically use something like lenses/path from Ramda: https://ramdajs.com/docs/#path

here you can read more: https://github.com/microsoft/TypeScript/issues/12290

Reply:
In general you could potentially make Sort<Transaction | Transaction["lastTransactionRef"]>

But it wont give you such string:

"lastTransactionRef.ordinal" :p